src/assets/js/protocol/protocol-footer.js

```diff
@@ -0,0 +1,21 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+(function() {
+    'use strict';
+
+    var _mqWide = matchMedia('(max-width: 479px)');
```

Comment:
We should add a basic feature detect for `matchMedia` here, just to stop older browsers throwing errors? I did one for the Menu component [here](https://github.com/mozilla/protocol/blob/master/src/assets/js/protocol/protocol-menu.js#L303-L307)

Doesn't need to be part of this PP, but we should file an issue to discuss where we should keep feature detects, as they could probably be consolidated in a library together.

Reply:
Moved the feature detect here and in the Menu into `protocol-supports.js`.